intl/msg_hash_us.h

```diff
@@ -560,6 +564,10 @@ MSG_HASH(
     MENU_ENUM_LABEL_VALUE_CHEEVOS_TEST_UNOFFICIAL,
     "Test Unofficial Achievements"
     )
+MSG_HASH(
+    MENU_ENUM_LABEL_VALUE_CHEEVOS_UNOFFICIAL_ENTRY,
+    "Unofficial"
+    )
```

Comment:
Is this the right way to add new strings? Do I need to stub in the other languages as well?

Reply:
In the past I used to did the stubs but in hindsight it was probably a bad call since it adds so much additional memory size to the binary for no real gain, so it's not necessary, no.